entrypoint/entrypoint.go

```diff
@@ -60,11 +60,12 @@ func New() (*cobra.Command, error) {
 
 		if scOpts.ResultsFile != "" {
 			var err error
-			out, err = os.Create(scOpts.ResultsFile)
+			resultsFilePath := fmt.Sprintf("%v/%v", opts.GithubWorkspace, scOpts.ResultsFile)
+			out, err = os.Create(resultsFilePath)
```

Comment:
non-blocking: by the time we get the results file, it should be complete and validated. That said, there's not much we can fix here (since the results file logic is in scorecard).

Leave a TODO here to take a look at this later.